lit_llama/model.py

```diff
@@ -35,14 +44,13 @@ def from_name(cls, name: str) -> Self:
 class LLaMA(nn.Module):
     def __init__(self, config: LLaMAConfig) -> None:
         super().__init__()
-        assert config.vocab_size is not None
-        assert config.block_size is not None
+        assert config.padded_vocab_size is not None
         self.config = config
 
-        self.lm_head = nn.Linear(config.n_embd, config.vocab_size, bias=False)
+        self.lm_head = nn.Linear(config.n_embd, config.padded_vocab_size, bias=False)
         self.transformer = nn.ModuleDict(
             dict(
-                wte=nn.Embedding(config.vocab_size, config.n_embd),
+                wte=nn.Embedding(config.padded_vocab_size, config.n_embd),
```

Comment:
The current default vocab size is a multiple of 64, so this should be save since the tensor size here won't actually be different to the shape in the checkpoint.